examples/todo-list/nuxt3-todo-list/sql-scripts/build-todo-app.sql

```diff
@@ -0,0 +1,133 @@
+------------------------------------------------------
```

Comment:
I think we can get rid of the files inside `sql-scripts` as long as we have the SQL statements in the readme.

Reply:
what if i used the supabase cli and then instead of sql-scripts, they would be under supabase/migrations/01-todo-example?

then instructions could encourage proper use of the cli, but also just reference the migration file if they want to hand-patch it.

Reply:
i will look at the next example - i can probably do that

Reply:
>what if i used the supabase cli and then instead of sql-scripts, they would be under supabase/migrations/01-todo-example? Then instructions could encourage proper use of the cli, but also just reference the migration file if they want to hand-patch it.

That could be fine.